go/integration_test/processor/calls_test.go

```diff
@@ -22,8 +22,50 @@ import (
 	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
 )
 
+const sufficientGas = tosca.Gas(1000000000000)
```

Comment:
```suggestion
const sufficientGas = tosca.Gas(1000_000_000_000)
```

This gas is double as the maximum allowed by the CT (to prevent memory expansions) Therefore using this
amount of gas may explore untested code regions.

https://github.com/Fantom-foundation/Tosca/blob/main/go/ct/st/state.go#L49

Reply:
Good catch, I changed it to the same value as `MaxGasUsedByCt`.